Sources/_CollectionsTestSupport/AssertionContexts/Assertions.swift

```diff
@@ -300,6 +300,22 @@ public func expectEqualElements<S1: Sequence, S2: Sequence>(
     message, trapping: trapping, file: file, line: line)
 }
 
+public func expectNotEqualElements<S1: Sequence, S2: Sequence>(
+  _ left: S1,
+  _ right: S2,
+  _ message: @autoclosure () -> String = "",
+  trapping: Bool = false,
+  file: StaticString = #file,
+  line: UInt = #line
+) where S1.Element == S2.Element, S1.Element: Equatable {
+  let left = Array(left)
+  let right = Array(right)
+  if !left.elementsEqual(right) { return }
+  _expectFailure(
+    "'\(left)' does have equal elements to '\(right)'",
+    message, trapping: trapping, file: file, line: line)
+}
+
```

Comment:
On second look, I recommend getting rid of this function -- it doesn't carry its weight.

```suggestion
```

(Also see the comment on `test_values_getter_not_equal_elements`.)